From f3492de8e77e5c339915edee94ac959b68998564 Mon Sep 17 00:00:00 2001
From: Christina Jacob <cjacob@marvell.com>
Date: Mon, 24 Jun 2019 13:17:54 +0530
Subject: [PATCH 322/386] octeontx2-af: sync ATF and Kernel firmware data
 structure.

Updated CGX_MAX to 4.
Renamed Macro RESERVED_MEM to FWDATA_RESERVED_MEM

Change-Id: Ic4201d851c499e2f9cb565540788b486d22fb8f2
Signed-off-by: Christina Jacob <cjacob@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/12079
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
[RH: Original patch taken from marvell 88F3720 board support SDK 10.0-PR2003]
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/af/rvu.h | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/drivers/net/ethernet/marvell/octeontx2/af/rvu.h b/drivers/net/ethernet/marvell/octeontx2/af/rvu.h
index 485ebc12a20e..2c09586795d6 100644
--- a/drivers/net/ethernet/marvell/octeontx2/af/rvu.h
+++ b/drivers/net/ethernet/marvell/octeontx2/af/rvu.h
@@ -386,9 +386,10 @@ struct rvu_fwdata {
 	u64 rclk;
 	u64 mcam_addr;
 	u64 mcam_sz;
-#define RESERVED_MEM 1024
-	u64 reserved[RESERVED_MEM];
-#define CGX_MAX         3
+#define FWDATA_RESERVED_MEM 1024
+	u64 reserved[FWDATA_RESERVED_MEM];
+	/* Do not add new fields below this line */
+#define CGX_MAX         4
 #define CGX_LMACS_MAX   4
 	struct cgx_lmac_fwdata_s cgx_fw_data[CGX_MAX][CGX_LMACS_MAX];
 };
-- 
2.17.1

